smart-contracts/cookbook/smart-contracts/deploy-basic/basic-hardhat.md

```diff
@@ -0,0 +1,120 @@
+- [Node.js](https://nodejs.org/en/download){target=\_blank} v22.13.1 or later installed.
+- Test tokens for gas fees, available from the [Polkadot faucet](https://faucet.polkadot.io/){target=\_blank}. See [Get Test Tokens](/smart-contracts/faucet/#get-test-tokens){target=\_blank} for a guide to using the faucet.
+- A wallet with a private key for signing transactions.
+
+## Set Up Your Project
+
+Use the following terminal commands to create a directory and initialize your Hardhat project inside of it:
+
+```bash
+mkdir hardhat-deployment
+cd hardhat-deployment
+npx hardhat --init
+```
+
+## Configure Hardhat
+
+Open `hardhat.config.js` and add `polkadotHubTestnet` to the `networks` configuration as highlighted in the following example code:
+
+```javascript title='hardhat.config.js' hl_lines='39-43'
+--8<-- 'code/smart-contracts/cookbook/smart-contracts/deploy-basic-contract/basic-hardhat/hardhat.config.js'
+```
+
+!!! tip
+    Learn how to use Hardhat's [Config Variables](https://hardhat.org/docs/learn-more/configuration-variables){target=\_blank} to handle your private keys securely.
+
+## Create the Contract
+
+Follow these steps to create your smart contract:
+
+1. Delete the default contract file(s) in the `contracts` directory.
+
+2. Create a new file named `Storage.sol` inside the `contracts` directory.
+
+3. Add the following code to create the `Storage.sol` smart contract:
+    ```solidity title="Storage.sol"
+    --8<-- 'code/smart-contracts/cookbook/smart-contracts/deploy-basic-contract/basic-hardhat/Storage.sol'
+    ```
+
+## Compile the Contract
+
+Compile your `Storage.sol` contract using the following command:
+
+```bash
+npx hardhat compile
+```
+
+You will see a message in the terminal confirming the contract was successfully compiled, similar to the following:
+
+--8<-- 'code/smart-contracts/cookbook/smart-contracts/deploy-basic-contract/basic-hardhat/compile-output.html'
+
+## Set Up Deployment
+
+Follow these steps to prepare for contract deployment:
+
+1. Delete the default file(s) inside the `ignition/modules` directory.
+
+2. Create a new file named `Storage.ts` inside the `ignition/modules` directory.
+
+3. Open `ignition/modules/Storage.ts` and add the following code to create your deployment module:
+
+    ```typescript title="ignition/modules/Storage.ts"
+    --8<-- 'code/smart-contracts/cookbook/smart-contracts/deploy-basic-contract/basic-hardhat/storage.ts'
+    ```
+
+## Deploy the Contract
+
+Deploy your contract to Polkadot Hub TestNet using the following command:
+
+```bash
+npx hardhat ignition deploy ignition/modules/Storage.ts --network polkadotHubTestnet 
+```
+
+Congratulations! You've now deployed a basic smart contract to Polkadot Hub TestNet using Hardhat. Consider the following resources to build upon your progress.
+
+## Where to Go Next
+
+<div class="grid cards" markdown>
+
+-   <span class="badge guide">Guide</span> __Verify Your Contract__
```

Comment:
Looks like this verify contract guide isn't making the cut for the MVP either, so can you please delete it?